src/app/components/Account/RuntimeAccountDetailsView.tsx

```diff
@@ -76,12 +76,7 @@ export const RuntimeAccountDetailsView: FC<RuntimeAccountDetailsViewProps> = ({
         <AccountAvatar account={account} />
       </StyledListTitleWithAvatar>
       <dd>
-        <AccountLink
-          showAddressAsName
```

Comment:
@donouwens 
Just to confirm we want to show duplicated names in contracts? 

![Screenshot from 2025-01-15 10-42-20](https://github.com/user-attachments/assets/7a2bde49-b03f-4c2c-b4ac-d5daead763ec)
![Screenshot from 2025-01-15 10-42-07](https://github.com/user-attachments/assets/596b8a90-ed0a-4378-900e-3d5743978c2b)


instead of 

![Screenshot from 2025-01-15 10-45-43](https://github.com/user-attachments/assets/f70ab368-de2e-40aa-a4cb-f2545033a165)

Reply:
For context, here is the link: https://pr-1687.oasis-explorer.pages.dev/mainnet/sapphire/address/0x6665a6Cae3F52959f0f653E3D04270D54e6f13d8

Reply:
Here is another link to another address page, which is not a token: https://pr-1687.oasis-explorer.pages.dev/testnet/pontusxtest/address/0x938224aC9e4832C517818B147BB1f6b10ADdCd26

Reply:
Maybe we should hide the name selectively just for token contracts?

Reply:
Updated behavior so that in account details, we don't show the account name if we also have a token name, but show it otherwise.